superset-frontend/src/dashboard/components/gridComponents/ChartHolder.jsx

```diff
@@ -159,9 +171,14 @@ class ChartHolder extends React.Component {
     });
   }
 
+  handleFullSize() {
+    const flag = !this.state.isFullSize;
```

Comment:
what is this `flag` for? Can we name it something more explicit like `newIsFullSizeState`?

Reply:
or `toggledFullSize`

Reply:
flag variable deleted c0526bd607a84ebc9e32f7d4676af14a06a2624b